### PR TITLE
add mode selection and correct some UX problems

### PR DESCRIPTION
**Pull request by ghemDD**

### CODE REVIEW

File: app/src/main/java/ch/epfl/sdp/blindwar/domain/game/GameSolo.kt [hunk 1]
```diff
@@ -0,0 +1,72 @@
+package ch.epfl.sdp.blindwar.domain.game
+
```
Comment (marluxiaboss) [inline app/src/main/java/ch/epfl/sdp/blindwar/domain/game/GameSolo.kt:2]: Why is there a whole class commented out ?  You should remove it if it is not necessary.

File: app/src/main/java/ch/epfl/sdp/blindwar/ui/StatisticsActivity.kt [hunk 2]
```diff
@@ -1,28 +1,77 @@
...
+        val correctView = findViewById<TextView>(R.id.correctNumberView)
+        correctView.text = "correctView"
+        val wrongView = findViewById<TextView>(R.id.wrongNumberView)
+        wrongView.text = "wrongView"
+        val correctPercent = findViewById<TextView>(R.id.correctnessPercentView)
+        correctPercent.text = "correctPercent"
+        val wrongPercent = findViewById<TextView>(R.id.wrongPercentView)
+        wrongPercent.text = "wrongPercent"
+
+        databaseReference.addValueEventListener(object : ValueEventListener {
```
Comment (marluxiaboss) [inline app/src/main/java/ch/epfl/sdp/blindwar/ui/StatisticsActivity.kt:53]: You can probably use the "addUserListener" function I created for this purpose in UserDatabase.

**Review by marluxiaboss — APPROVED** [review]
Good additions for this sprint! I also liked that you left out some part for the next sprint so that you were able to make a PR for this sprint.

**Review by JorisMonnet — APPROVED** [review]
as @marluxiaboss said, the entire commented class should not be on main

Kdoc is missing on a lot of your new functions

**Review by kapapp — COMMENTED** [review]
Consider adding commentary to DemoFragmentTest, also layout is not optimised for all screen sizes

**Pull request merged**